LICENSE

```diff
@@ -0,0 +1,15 @@
+The ISC License
```

Comment:
Why ISC and not MIT?

Reply:
Cause it said so in the readme! 🤷‍♂️

Reply:
I think that's npm's default. No idea why, really. Will look into that.